### PR TITLE
Update log information

### PR DESCRIPTION
**Pull request by ricardoas**

### CODE REVIEW

File: networking_oneview/ml2/drivers/oneview/common.py [hunk 1]
```diff
@@ -742,6 +743,8 @@ def check_uplinkset_types_constraint(oneview_client, uplinkset_mappings):
     It is only possible to map one provider to at the most one uplink
     of each type.
     """
+    LOG.info("Check if a provider has two mappings for the same LIG with"
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/common.py]: This needs a space at the end of the first part of the string.

File: networking_oneview/ml2/drivers/oneview/common.py [hunk 2]
```diff
@@ -799,6 +803,7 @@ def check_unique_lig_per_provider_constraint(uplinkset_mappings):
 
 
 def delete_outdated_flat_mapped_networks(flat_net_mappings):
+    LOG.info("Synchronizing flat network mappings")
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/common.py]: Why not Synchronize, to keep the same pattern as the other Logs.

File: networking_oneview/ml2/drivers/oneview/synchronization.py [hunk 3]
```diff
@@ -43,6 +43,7 @@ def start(self):
 
     @common.oneview_reauth
     def synchronize(self):
+        LOG.info("Starting synchronization mechanism.")
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/synchronization.py:46]: Start

File: networking_oneview/ml2/drivers/oneview/synchronization.py [hunk 4]
```diff
@@ -61,6 +63,7 @@ def get_oneview_network(self, oneview_net_id):
             LOG.error(err)
 
     def create_oneview_networks_from_neutron(self):
+        LOG.info("Synchronizing Neutron networks not in OneView")
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/synchronization.py]: ditto

File: networking_oneview/ml2/drivers/oneview/synchronization.py [hunk 5]
```diff
@@ -90,6 +93,7 @@ def create_oneview_networks_from_neutron(self):
             self.neutron_client.network.create(session, network_dict)
 
     def synchronize_uplinkset_from_mapped_networks(self):
+        LOG.info("Synchronizing OneView uplinksets")
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/synchronization.py]: ditto

File: networking_oneview/ml2/drivers/oneview/synchronization.py [hunk 6]
```diff
@@ -105,6 +109,7 @@ def synchronize_uplinkset_from_mapped_networks(self):
                             'physical_network'))
 
     def delete_unmapped_oneview_networks(self):
+        LOG.info("Synchronizing outdated networks in OneView")
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/synchronization.py]: ditto

File: networking_oneview/ml2/drivers/oneview/synchronization.py [hunk 7]
```diff
@@ -209,6 +214,7 @@ def recreate_connection(self):
         Calls method to fix critical connections in the Server Profile that
         will be used.
         """
+        LOG.info("Synchronizing connections in OneView Server Profiles")
```
Comment (stenioaraujo) [inline networking_oneview/ml2/drivers/oneview/synchronization.py]: ditto

**Review by stenioaraujo — CHANGES_REQUESTED** [review]

File: networking_oneview/ml2/drivers/oneview/common.py (hunk 2 above)
Comment (hugonicodemos) [reply networking_oneview/ml2/drivers/oneview/common.py]: +1

File: networking_oneview/ml2/drivers/oneview/synchronization.py (hunk 3 above)
Comment (hugonicodemos) [reply networking_oneview/ml2/drivers/oneview/synchronization.py:46]: +1

**Review by hugonicodemos — APPROVED** [review]

**Review by stenioaraujo — APPROVED** [review]

**Pull request merged**